strapi/strapi
# feat(content-releases): Edit Release

### What does it do?

Enable the Edit button in the Details page to change the release name

### Why is it needed?

We need a way to edit the release name

### How to test it?

You can go to the Release page and create new Release
- then in the ReleaseDetails page with some data mocked, you can click on the three dots button and see the Edit button
- when you click on the edit button the Dialog with the actual release title (a mocked value) is shown and you can change the value and save it
